src/KeyVault/KeyVault/Commands/Certificate/Policy/SetAzureKeyVaultCertificatePolicy.cs

```diff
@@ -261,7 +262,7 @@ public class SetAzureKeyVaultCertificatePolicy : KeyVaultCmdletBase
         /// Elliptic Curve Name of the key
         /// </summary>
         [Parameter(Mandatory = false,
-                   ValueFromPipelineByPropertyName = true,
+                   ValueFromPipelineByPropertyName = false,
```

Comment:
```suggestion
```
Remove this attribute directly

Reply:
Updated this in the code, removed the instances of `ValueFromPipelineByPropertyName` where it was set to `false`